PhysicsTools/PatAlgos/plugins/PATTriggerProducer.h

```diff
@@ -129,6 +129,7 @@ namespace pat {
         bool lastFilter;
         bool l3Filter;
       };
+      ModuleLabelToPathAndFlags() {}
```

Comment:
it seems more appropriate to follow the reason for the error and change the `empty_`  initialization
```diff
@@ -142,7 +142,7 @@ namespace pat {
         map_[filter].push_back(PathAndFlags(path, pathIndex, lastFilter, l3Filter));
       }
       std::map<std::string, std::vector<PathAndFlags> > map_;
-      const std::vector<PathAndFlags> empty_;
+      const std::vector<PathAndFlags> empty_ = {};
     };
     ModuleLabelToPathAndFlags moduleLabelToPathAndFlags_;
   };
```

Reply:
clang compiles after just the change above in my case.

Reply:
thanks @slava77 , I will update the PR

Reply:
done